generator/docs/NullOperator.md

```diff
@@ -0,0 +1,168 @@
+# `null`, presence, absence and the empty set.
+
+The `null` operator in a profile, expressed as `"is": "null"` or the negated equivalent has several meanings. It can mean (and emit the behaviour) as described below:
+
+| Absence / Presence<br />Described in other technologies as nullability | Field value<br /> |
+| ---- | ---- |
+| _null operator omitted_<br /> **The default**. The field's value may be absent or present | `is null`<br />The field will have _no value_ |
+| `not(is null)`<br />The field's value must be present | `not(is null)`<br />The field must have a value |
+
+Therefore the null operator can:
+- `not(is null)` express fields that must have a value (otherwise known as a non-nullable field)
+- `is null` express fields as having no value (otherwise known as setting the value to `null`)
+- _By omitting the constraint_: express fields as allowing either having a value or the absent of one (otherwise known as a nullable field)
+
```

Comment:
Last bullet point is a little unclear, perhaps: "express fields as allowing a value to be present or absent (otherwise..)"

Reply:
Updated to:
> _By omitting the constraint_: express fields as permitting absence or presence of a value (otherwise known as a nullable field)